collector/meminfo_numa_linux.go

```diff
@@ -33,8 +33,13 @@ const (
 	memInfoNumaSubsystem = "memory_numa"
 )
 
-type meminfoKey struct {
-	metricName, numaNode string
+var nodeRE = regexp.MustCompile(`.*devices/system/node/node([0-9]*)`)
```

Comment:
This is a global variable visible in all files of the collector package. Can we maybe use a more specific name?